cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc

```diff
@@ -301,7 +301,7 @@ bool SubstraitToVeloxPlanValidator::validateCast(
       return false;
     case TypeKind::TIMESTAMP:
       // Only support cast timestamp to date
```

Comment:
Comment need change.
I remember that cast timestamp to varchar will get incorrect results. Can the velox changes here fix it? @PHILO-HE

Reply:
@zml1206, thanks for your comment! Yes, Spark UTs have uncovered some wrong result issue. I am still trying to fix all issues.